database/seeders/DatabaseSeeder.php

```diff
@@ -12,5 +12,6 @@ class DatabaseSeeder extends Seeder
     public function run(): void
     {
         $this->call(AirlineSeeder::class);
+        $this->call(CitySeeder::class);
```

Comment:
```suggestion
        $this->call([
            AirlineSeeder::class,
            CitySeeder::class
        ]);
```
You can do this instead 😉